packages/core/src/operator/operator-runtime/bootOperatorRuntime.ts

```diff
@@ -111,6 +116,11 @@ export const bootOperatorRuntime = async (
 
         // Check if the referee has been upgraded to V2
         const refereeIsV2 = await checkRefereeBulkSubmissionCompatible();
+
+        // Get the latest challenge from the RPC
+        const [latestChallengeNumber, latestChallenge] = await getLatestChallenge();
```

Comment:
This was called in both paths below, so I moved it up to eliminate the duplication.